examples/pastebin/src/paste_id.rs

```diff
@@ -35,11 +35,7 @@ impl<'a> fmt::Display for PasteID<'a> {
 
 /// Returns `true` if `id` is a valid paste ID and `false` otherwise.
 fn valid_id(id: &str) -> bool {
-    id.chars().all(|c| {
-        (c >= 'a' && c <= 'z')
-            || (c >= 'A' && c <= 'Z')
-            || (c >= '0' && c <= '9')
-    })
+    id.chars().all(|c| c.is_ascii_alphanumeric())
```

Comment:
Does `all(char::is_ascii_alphanumeric)` work here (and elsewhere in this changeset)?

Reply:
Unfortunately not, I think because the string is passed as reference.
As far as I can tell this is the only place where it would be applicable in the changeset.
```
error[E0631]: type mismatch in function arguments
  --> examples/pastebin/src/paste_id.rs:38:20
   |
38 |     id.chars().all(char::is_ascii_alphanumeric)
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |                    |
   |                    expected signature of `fn(char) -> _`
   |                    found signature of `for<'r> fn(&'r char) -> _`
```